blobfuse/src/BlobfuseGlobals.cpp

```diff
@@ -29,22 +29,27 @@ AUTH_TYPE get_auth_type(std::string authStr)
 {
     if(!authStr.empty()) {
         std::string lcAuthType = to_lower(authStr);
-        if (lcAuthType == "msi") {
+        syslog(LOG_DEBUG, "Auth type set as %s", lcAuthType.c_str());
+        int lcAuthTypeSize = (int)lcAuthType.size();
+         // sometimes an extra space or tab sticks to authtype thats why this size comparison, it is not always 3 lettered
+        if(lcAuthTypeSize > 0 && lcAuthTypeSize < 5) 
```

Comment:
Even if do the size check to safeguard extra space and tab, below we are comparing the data without handling the same. We should rather trim the value first and then do the comparisons.

Reply:
done trimmed.